node/status_node_services.go

```diff
@@ -441,9 +441,9 @@ func (b *StatusNode) appmetricsService() common.StatusService {
 	return b.appMetricsSrvc
 }
 
-func (b *StatusNode) walletService(accountsFeed *event.Feed, openseaAPIKey string) common.StatusService {
+func (b *StatusNode) walletService(accountsFeed *event.Feed, openseaAPIKey string, accountsDB *accounts.Database) common.StatusService {
 	if b.walletSrvc == nil {
-		b.walletSrvc = wallet.NewService(b.appDB, b.rpcClient, accountsFeed, openseaAPIKey)
+		b.walletSrvc = wallet.NewService(b.appDB, b.rpcClient, accountsFeed, openseaAPIKey, b.gethAccountManager, accountsDB)
```

Comment:
nit: appDB is already a parameter, you don t really need to add accountsDB which can be rebuild from appDB?

Reply:
aaah ok let me check :)